sdk/metrics/src/test/java/io/opentelemetry/sdk/metrics/aggregator/AggregatorFactoryTest.java

```diff
@@ -123,4 +124,73 @@ void getSumAggregatorFactory() {
                     InstrumentValueType.DOUBLE)))
         .isInstanceOf(DoubleSumAggregator.class);
   }
+
+  @Test
+  void getHistogramAggregatorFactory() {
+    AggregatorFactory histogram =
+        AggregatorFactory.histogram(new double[] {1.0}, AggregationTemporality.DELTA);
+    assertThat(
+            histogram.create(
+                Resource.getDefault(),
+                InstrumentationLibraryInfo.empty(),
+                InstrumentDescriptor.create(
+                    "name",
+                    "description",
+                    "unit",
+                    InstrumentType.VALUE_RECORDER,
+                    InstrumentValueType.LONG)))
+        .isInstanceOf(DoubleHistogramAggregator.class);
+    assertThat(
+            histogram.create(
+                Resource.getDefault(),
+                InstrumentationLibraryInfo.empty(),
+                InstrumentDescriptor.create(
+                    "name",
+                    "description",
+                    "unit",
+                    InstrumentType.VALUE_RECORDER,
+                    InstrumentValueType.DOUBLE)))
+        .isInstanceOf(DoubleHistogramAggregator.class);
+
+    assertThat(
+            histogram
+                .create(
+                    Resource.getDefault(),
+                    InstrumentationLibraryInfo.empty(),
+                    InstrumentDescriptor.create(
+                        "name",
+                        "description",
+                        "unit",
+                        InstrumentType.VALUE_RECORDER,
+                        InstrumentValueType.LONG))
+                .isStateful())
+        .isFalse();
+    assertThat(
+            AggregatorFactory.histogram(new double[] {1.0}, AggregationTemporality.CUMULATIVE)
+                .create(
+                    Resource.getDefault(),
+                    InstrumentationLibraryInfo.empty(),
+                    InstrumentDescriptor.create(
+                        "name",
+                        "description",
+                        "unit",
+                        InstrumentType.VALUE_RECORDER,
+                        InstrumentValueType.DOUBLE))
+                .isStateful())
+        .isTrue();
+
+    Assertions.assertThrows(
```

Comment:
Oh looks like we have a few left in `MetricDataTest` but we've switched to just using assertj, can you use `assertThatThrownBy(() ->).isInstanceOf()`? And we like to follow up with `hasMessage` or `hasMessageContaining` with a light assertion on the message.